plugins/GSdx/GSDeviceOGL.cpp

```diff
@@ -572,9 +572,10 @@ GLuint GSDeviceOGL::CreateSampler(bool bilinear, bool tau, bool tav)
 	gl_SamplerParameterf(sampler, GL_TEXTURE_MIN_LOD, 0);
 	gl_SamplerParameterf(sampler, GL_TEXTURE_MAX_LOD, 6);
 
-	if (GLLoader::found_GL_EXT_texture_filter_anisotropic && !!theApp.GetConfig("AnisotropicFiltering", 0) && !theApp.GetConfig("paltex", 0)) {
+	if (GLLoader::found_GL_EXT_texture_filter_anisotropic && (int)theApp.GetConfig("MaxAnisotropy", 0)>1 && !theApp.GetConfig("paltex", 0)) 
```

Comment:
It would be better this way:

int anisotropy = theApp.GetConfig("MaxAnisotropy", 0);
if (GLLoader::found_GL_EXT_texture_filter_anisotropic && anisotropy && !theApp.GetConfig("paltex", 0))
Then correct the indentation of the code